kernel-cert-script

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+op=$(echo "$0" | sed -e "s/.*-//")
+
+while true ; do
+    case $1 in
+	--kmp)
+	    kmp="$2"
+	    shift 2
+	    ;;
+	--certs)
+	    certs="$2"
+	    shift 2
+	    ;;
+	*) break
+	    ;;
+    esac
+done
+
+run_mokutil () {
+    [ -z "$KERNEL_PACKAGE_SCRIPT_DEBUG" ] || echo mokutil "$@" > /dev/kmsg
+    mokutil "$@"
+}
+
+[ -z "$KERNEL_PACKAGE_SCRIPT_DEBUG" ] || echo "cert $op" kmp: "$kmp" certs: "$certs" > /dev/kmsg
+
+mok_rc=0
+
+case $op in
+    pre)
+	;;
+    preun)
+	if command -v mokutil >/dev/null; then
```

Comment:
So long as there is no clause that does something else.

Also do we want to adhere to bash-free syntax or just make the script depend on bash?

Reply:
-v is documented by command (1p) so I suppose it is standardized

If somebody complains that their limited shell does not have it we can solve it when that happens.

Note that the kernel uses this exact code and nobody complained.

Reply:
There is the argument check that whines about wrong argument which should be executed regardless of having mokutil so it should not be put outside of the case.

It can be made into a function/variable to make changing the -v easier in the future but it's not a big wine because it still has to be tested in each branch of the case anyway.

Reply:
I checked for bashism because you started the script with `#! /bin/sh`. I'm fine with using `#! /bin/bash` because `weak-modules2` needs it anyway. You're right wrt POSIX, so `check_bashisms` is probably simply wrong here.